packages/crypto/src/hashing.ts

```diff
@@ -47,18 +47,24 @@ async function hashTransaction(tx: TxBody, encoding: 'bytes', includeSign?: bool
 async function hashTransaction(tx: TxBody, encoding = 'base64', includeSign = true): Promise<Buffer | string> {
     // check amount format
     tx.amount = '' + tx.amount;
```

Comment:
I meant to suggest to also remove this line. It shouldn't be necessary.
```suggestion
```

Reply:
if I remove 49 line and add 50 line toString()
Is this code make error when tx.amount value is null?

Reply:
remove 49 line and add 50 line 'toString()'

(property) TxBody.amount?: string | number | JSBI | Record<string, any> | undefined
Object is possibly 'undefined'. 😢

Reply:
ah ok. then never mind, I might change that later.